packages/firestore/src/remote/serializer.ts

```diff
@@ -83,6 +83,9 @@ const OPERATORS = (() => {
   return ops;
 })();
 
+// A RegExp matching ISO 8601 UTC timestamps with optional fraction.
+const ISO_REG_EXP = new RegExp(/^\d{4}-\d\d-\d\dT\d\d:\d\d:\d\d(?:\.(\d+))?Z$/);
```

Comment:
I originally moved everything connected to parsing ISO strings here to avoid exposing this functionality in the public API. I guess it was possible to leave it on the Timestamp and not declare it in `index.d.ts` file (at least I assume so), but on the other hand, that functionality is only used in the serializer. Feel free to push back on this.

Reply:
Yeah, it would be fine to have it in Timestamp but not expose it in the .d.ts. Though we would also want to prefix the APIs with '_' to ensure they're minified.  That said, it seems like ISO timestamp conversion is specific to the web serializer implementation (we don't have iso timestamp conversion in the iOS / Android Timestamp.java implementation) and so keeping it in the serializer sounds good to me.